src/node-saml/saml.ts

```diff
@@ -1175,11 +1175,20 @@ class SAML {
             // if attributes has no AttributeValue child, continue
             return;
           }
+          const name = attribute.$.Name;
           const value = attribute.AttributeValue;
+
+          // If any property is already present in profile and is also present
+          // in attributes, then skip the one from attributes. Handle this
+          // conflict gracefully without returning any error
+          if (Object.prototype.hasOwnProperty.call(profile, name)) {
```

Comment:
Do you think there is value is putting all these values in a child object on the profile in addition to cloning them directly to the profile object where there is no conflict? This way, nothing is lost, but we also don't break existing behavior.

Reply:
Yes. I think putting all attributes in separate child property would be useful. And only clone the properties that do not exist on profile.

Reply:
Update PR with the requested changes